mypyc/lib-rt/pythonsupport.h

```diff
@@ -401,6 +401,8 @@ _CPyObject_HasAttrId(PyObject *v, _Py_Identifier *name) {
     _PyObject_CallMethodIdObjArgs((self), (name), NULL)
 #define _PyObject_CallMethodIdOneArg(self, name, arg) \
     _PyObject_CallMethodIdObjArgs((self), (name), (arg), NULL)
```

Comment:
Maybe we should remove `_PyObject_CallMethodIdNoArg` and `_PyObject_CallMethodIdOneArg` here. `_PyObject_CallMethodIdObjArgs` is no longer exported in Python 3.13.

Reply:
Never mind, those are only used for `< 3.9` anyway.